Vagrantfile

```diff
@@ -10,6 +10,9 @@ Vagrant.configure(2) do |config|
   # For a complete reference, please see the online documentation at
   # https://docs.vagrantup.com.
 
+  # ssh agent forwarding
+  config.ssh.forward_agent = true
```

Comment:
Hi Kristen. Sorry, I just saw this now. So the reason for this change was that in order to use the Vagrant environment I kept getting error messages saying my keys were not verified and I couldn't push. So I got some outside advice to do ssh agent forwarding to fix this issue. It did fix the issue and I was able to push the commit.

The error:
Dalias-MacBook-Pro:gdi-website dalia$ git fetch upstream
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.